autogluon/task/text_classification/dataset.py

```diff
@@ -141,93 +138,7 @@ class CustomTSVClassificationTask(AbstractGlueTask):
-    keep_date_col : bool, default False
-        If True and `parse_dates` specifies combining multiple columns then
-        keep the original columns.
-    date_parser : function, optional
-        Function to use for converting a sequence of string columns to an array of
-        datetime instances.
-    dayfirst : bool, default False
-        DD/MM format dates, international and European format.
-    cache_dates : bool, default True
-        See <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv`_ .
-    iterator : bool, default False
-        See <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv`_ .
-    chunksize : int, optional
-        See <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv`_ .
-    compression : {{'infer', 'gzip', 'bz2', 'zip', 'xz', None}}, default 'infer'
-    thousands : str, optional
-        See <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv`_ .
-    decimal : str, default '.'
-        See <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv`_ .
-    lineterminator : str (length 1), optional
-        See <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv`_ .
-    quotechar : str (length 1), optional
-        See <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv`_ .
-    quoting : int or csv.QUOTE_* instance, default 0
-        See <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv`_ .
-    doublequote : bool, default ``True``
-        See <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv`_ .
-    escapechar : str (length 1), optional
-        See <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv`_ .
-    comment : str, optional
-        See <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv`_ .
-    encoding : str, optional
-        See <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv`_ .
-    dialect : str or csv.Dialect, optional
-        See <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv`_ .
-    error_bad_lines : bool, default True
-        See <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv`_ .
-    warn_bad_lines : bool, default True
-        See <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv`_ .
-    delim_whitespace : bool, default False
-        See <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html#pandas.read_csv`_ .
-    low_memory : bool, default True
```

Comment:
low_memory mode should always be False, it is harmful in many cases.

Reply:
Please comment in the updated version. Thanks

Reply:
I am saying that by default pd.read_csv sets low_memory=True, but you should explicitly default it to False in your code.